### ISSUE TITLE
Борт. Сигнализация на Юбилейном

### ISSUE THREAD

**Issue by AlmP-RUS** [metrostroi-repo/MetrostroiAddon#340]
На Юбилейном в реальной жизни на немоторных вагонах 2 лампы на бортах заместо трёх. Смею предположить, что нету зеленой лампы РП, поскольку вагон немоторный и силовой схемы как таковой не имеет

**Comment by AlmP-RUS** [comment]
Если в метрострое это уже реализовано, значит ~~дед опять забыл принять таблетки~~ я проглядел

**Comment by AlmP-RUS** [comment]
Да, я еще раз проехался на юбике, и на немоторном только желтая и белая лампа на бортах

**Comment by alexandr-sizov** [comment]
Баг.![Юбилейный_в_Невском_(3).jpg](https://user-images.githubusercontent.com/31768135/128185599-723f57fd-4c25-4df6-8e04-b1362079c943.jpg)

**Comment by CrIcKeT98** [comment]
Oldy не выдал мне исходников 720 и 722, а заниматься рекомпиляцией я не буду, там нормали съезжают и добавляются лишние разрезы на развёртке.. Поэтому пока что этот баг не будет исправлен
